components/match2/wikis/rocketleague/match_summary.lua

```diff
@@ -41,15 +42,15 @@ local Casters = Class.new(
 
 function Casters:addCaster(caster)
 	if Logic.isNotEmpty(caster) then
-		table.insert(self.casters, '[[' .. caster .. ']]')
+		table.insert(self.casters, Flags.Icon(caster['flag']) .. ' [[' .. caster['name'] .. ']]')
 	end
 	return self
 end
 
 function Casters:create()
 	return self.root
-		:wikitext('<b>Caster' .. (#self.casters > 1 and 's' or '') .. ':</b><br>')
-		:wikitext(table.concat(self.casters, ', '))
+		:wikitext('Caster' .. (#self.casters > 1 and 's' or '') .. ': ')
+		:wikitext(table.concat(self.casters, #self.casters > 2 and ', ' or ' & '))
```

Comment:
`{'Foo', 'Bar', 'Rath'}` would display `Foo, Bar, Rath` while `{'Foo', 'Bar'}` would display `Foo & Bar`. Is it really intended?

Reply:
I prefer the ampersand, it reads more like a casting-duo than a list that way

Reply:
My point was more that when there are three casters, there won't be an ampersand.
I would expect the output for 3 casters to be `Foo, Bar & Rath`, not `Foo, Bar, Rath`

Reply:
If the ampersand was there with the commas, I'd like there to be an Oxford comma, but it doesn't look like `listToText` supports that

Reply:
Oxford comma is generally not used outside of Academia, not used anywhere on Liquipedia to my knowledge and to my understanding it's mostly omitted when used with ampersand.

Reply:
I personally think that any list without an Oxford comma reads strangely, so I didn't put an "and"/ampersand at all